build.gradle

```diff
@@ -227,6 +242,13 @@ task renderRR() {
         } 
     }
 }
+
+task updateKeywords(type: JavaExec) {
```

Comment:
Since I do not use gradle, what does this do?

Maven is the main build engine. You changed some gradle build options. Does the maven build still run?

Reply:
It generates and prints the Source Code Text of the method `RelObjectName()` which you would manually insert/replace in the JSQL Grammar file.

This step is optional and does not affect the Maven build. It is executed manually and on demand only.

Although the long term goal was to have a mechanism inside the build tool (Gradle and/or Maven), which during the build:

1) analyses the KeyWords automatically
2) modifies the Grammar file (methods `RelObjectName()`  and friends)
3) builds the Parser with the modified Grammar
4) runs the Keyword Tests

Reply:
I think this might also be implemented in a mojo in Maven, but that might be not worth the effort.